src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -461,10 +461,19 @@ class HttpGoogleServicesDAO(implicit val system: ActorSystem, implicit val mater
   }
 
   /** Fetch the latest price list from Google. Returns only the subset of prices that we find we have use for. */
-  def fetchPriceList(implicit executionContext: ExecutionContext): Future[GooglePriceList] = {
-    val httpReq = Get(FireCloudConfig.GoogleCloud.priceListUrl)
-
-    unAuthedRequestToObject[GooglePriceList](httpReq)
+  //Why is this a val? Because the price lists do not change very often. This prevents making an HTTP call to Google
+  //every time we want to calculate a cost estimate (which happens extremely often in the Terra UI)
+  //Because the price list is brittle and Google sometimes changes the names of keys in the JSON, there is a
+  //default cached value in configuration to use as a backup. If we fallback to it, the error will be logged
+  //but users will probably not notice a difference. They're cost *estimates*, after all.
+  val fetchPriceList: Future[GooglePriceList] = {
```

Comment:
sorry, one more thought: should this be `lazy`?

If it's `lazy`, then we'd avoid querying Google every time a new `HttpGoogleServicesDAO` is constructed, for instance if someone wrote a set of unit tests that constantly make new daos.

If it's NOT `lazy`, then by eagerly querying Google on construction of the DAO, we avoid any potential pauses or race conditions when the price list is needed.

I lean towards making it lazy.